services/github/src/fake-payloads.js

```diff
@@ -745,7 +745,7 @@ export const getEventPayload = function getEventPayload(type, action, organizati
     mapping[type]
       .replaceAll('$ORGANIZATION', organization)
```

Comment:
`organization` and `branch` were validated with schema, but `branch` was coming from a free-form `overrides` param